### ISSUE TITLE
Fix asset fieldtype in Grid Table mode

### ISSUE THREAD

**Issue by mikemartin** [statamic/cms#6832]
### Bug description

When using the asset fieldtype in table mode, the asset preview shrinks to the point that it's not visible.

<img width="844" alt="Screen Shot 2022-10-05 at 2 48 42 pm" src="https://user-images.githubusercontent.com/414211/193977878-5409717e-585a-4c9c-9d1d-bc352c0d558b.png">
<img width="831" alt="Screen Shot 2022-10-05 at 2 48 25 pm" src="https://user-images.githubusercontent.com/414211/193977879-2ee7eb67-9738-4805-bbe1-fba677cc2284.png">

Here's how it looks when stacked. Note that I have the filename hidden.
<img width="833" alt="Screen Shot 2022-10-05 at 2 50 32 pm" src="https://user-images.githubusercontent.com/414211/193977972-cf5b046f-25f5-4597-bfb5-4e6c0564da6b.png">

### How to reproduce

Add an asset field in a grid with 20% width.

```
-
        handle: key_points
        field:
          type: grid
          display: 'Key Features'
          fields:
            -
              handle: icon
              field:
                mode: list
                container: assets
                folder: streamline
                restrict: false
                allow_uploads: true
                max_files: 1
                display: Icon
                type: assets
                icon: assets
                listable: hidden
                instructions_position: above
                read_only: false
                show_filename: false
                width: 20
            -
              handle: title
              field:
                type: text
                localizable: true
                width: 40
            -
              handle: point
              field:
                type: textarea
                localizable: true
                width: 40
          instructions_position: above
          listable: hidden
          reorderable: true
          mode: stacked
          add_row: 'Add Feature'
```

### Environment

```yaml
Environment
Application Name: Burlingame
Laravel Version: 9.32.0
PHP Version: 8.0.21
Composer Version: 2.4.0
Environment: local
Debug Mode: ENABLED
URL: burlingame.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 8
Antlers: runtime
Version: 3.3.42 PRO

Statamic Addons
aerni/livewire-forms: 6.2.0
jonassiewertsen/statamic-livewire: 2.9.0
mikemartin/helpscout-beacon: 1.0.2
rias/statamic-color-swatches: 2.0.5
spatie/statamic-responsive-images: 2.14.0
swiftmade/statamic-clear-assets: 1.1.0
webographen/statamic-widget-continue-editing: 1.0.1
withcandour/aardvark-seo: 2.0.28
```

### Installation

Other (please explain)

**Comment by jacksleight** [comment]
Same issue when the fields are set to < 100% wide. With single asset fields I often use `grid` mode to provide a nice big preview of the image, eg:

![Screenshot 2022-10-14 at 14 17 40](https://user-images.githubusercontent.com/126740/195856999-8bfddcbd-63b2-45c4-af18-fac33286743d.png)

But now that looks like this:

![Screenshot 2022-10-14 at 14 16 46](https://user-images.githubusercontent.com/126740/195857061-21af08de-7761-4624-a2ee-78ce06405e6c.png)

**Comment by arthurperton** [comment]
Hi @mikemartin, I wonder if your issue was solved by #6888?

**Comment by jasonvarga** [comment]
Feel free to reopen if you don't consider it fixed.
